app/src/main/res/layout/fragment_scheduler.xml

```diff
@@ -8,189 +8,272 @@
+                        fab:fab_size="mini"/>
+
+                    <com.github.clans.fab.FloatingActionButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:src="@drawable/ic_smile_24dp"
+                        app:rippleColor="@color/color_white"
+                        bind:ignore="VectorDrawableCompat"
+                        fab:fab_colorNormal="@color/color_white"
+                        fab:fab_label="@string/tilte_finished"
+                        fab:fab_size="mini"/>
 
-            <com.github.clans.fab.FloatingActionButton
+                    <com.github.clans.fab.FloatingActionButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:src="@drawable/ic_broken_heart_24dp"
+                        bind:ignore="VectorDrawableCompat"
+                        fab:fab_colorNormal="@color/color_white"
+                        fab:fab_label="@string/title_cancel"
+                        fab:fab_size="mini"/>
+
+                    <com.github.clans.fab.FloatingActionButton
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:src="@drawable/ic_waiting_room_24_dp"
+                        bind:ignore="VectorDrawableCompat"
+                        fab:fab_colorNormal="@color/color_white"
+                        fab:fab_label="@string/title_waiting"
+                        fab:fab_size="mini"/>
+                </com.github.clans.fab.FloatingActionMenu>
+            </android.support.design.widget.CoordinatorLayout>
+
+            <android.support.design.widget.NavigationView
                 android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:src="@drawable/ic_waiting_room_24_dp"
-                bind:ignore="VectorDrawableCompat"
-                fab:fab_colorNormal="@color/color_white"
-                fab:fab_label="@string/title_waiting"
-                fab:fab_size="mini"/>
-        </com.github.clans.fab.FloatingActionMenu>
-    </android.support.design.widget.CoordinatorLayout>
+                android:layout_height="match_parent"
+                android:layout_gravity="start"
+                android:background="@color/color_white">
+
+                <android.support.v4.widget.NestedScrollView
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    >
+
+                    <LinearLayout
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:orientation="vertical"
+                        android:paddingLeft="@dimen/dp_10"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:paddingStart="@dimen/dp_10"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)